ci/pipeline.yml

```diff
@@ -112,14 +123,14 @@ jobs:
       text: |
         :x: FAILED to deploy logs-opensearch on development
         <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
-      channel: ((slack-failure-channel))
```

Comment:
I know Van asked us to change this, but I don't really agree. I think it's good for failures to be visible

Reply:
Maybe a way for us to differentiate development vs everyday?

Reply:
If we really want to do that, we should create a credhub var called `slack-dev-channel` then use that here instead of hardcoding the channel name